authLdap.php

```diff
@@ -412,33 +414,32 @@ function authLdap_get_uid($username)
 }
 
 /**
- * Get the user's current role
+ * Get the user's current roles
  *
  * Returns empty string if not found.
  *
  * @param int $uid wordpress user id
- * @return string role, empty if none found
+ * @return array roles, empty if none found
  */
-function authLdap_user_role($uid)
+function authLdap_user_roles($uid)
 {
     global $wpdb;
 
     if (!$uid) {
-        return '';
+        return array( );
```

Comment:
Please replace ```array()``` with ```[]``` - This library only supports PHP5.4 and above. And even that is by now outdated :frowning: (http://php.net/supported-versions.php)